packages/core/src/saml-applications/libraries/secrets.ts

```diff
@@ -0,0 +1,34 @@
+import { generateStandardId } from '@logto/shared';
+
+import type Queries from '#src/tenants/Queries.js';
+
+import { generateKeyPairAndCertificate } from './utils.js';
+
+export const createSamlApplicationSecretsLibrary = (queries: Queries) => {
```

Comment:
Can we just keep one single SAML application library?

Reply:
Good call, this is renamed in [the upcoming PR](https://github.com/logto-io/logto/pull/6827/files#diff-48cf87bd12883a7f28c7f9d2c4b28e2c841075bc068e853dca2b1dfa011d8108).